README.md

```diff
@@ -65,20 +65,22 @@ Installation
 
 1. Developer unlock your chromebook
 2. Open crosh shell
+3. Use sudo or get root
 
 ```
-cd ~/Downloads
-wget https://github.com/nmilosev/crouton-fedora-wayland/archive/master.tar.gz -O crouton-fedora-wayland.tar.gz
-tar xvf crouton-fedora-wayland.tar.gz && rm crouton-fedora-wayland.tar.gz
+
+cd /usr/local
+sudo curl -L -#  https://github.com/nmilosev/crouton-fedora-wayland/archive/master.tar.gz -o crouton-fedora-wayland.tar.gz
+sudo curl -L -#  https://github.com/boydkelly/crouton-fedora-wayland/archive/F30.tar.gz -o crouton-fedora-wayland.tar.gz
```

Comment:
All is looking good! Not sure about this line, can we just pull the master from this repo?

Thanks!